tests/cypress/e2e/registration/registrationModalRedirect.cy.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+
+describe('Registration modal redirect flow', { testIsolation: false }, () => {
+  const devices = ['desktop', 'mobile']
+
+  devices
+    .filter((device) => Cypress.env('devices')[`${device}`])
+    .forEach((device) => {
+      context(device, Cypress.env('resolution')[`${device}`], () => {
+
```

Comment:
beforeEach(() => {
cy.hideNavbar(device)
})

Use cy.hideNavbar(device) before each test in this file. Use this function.